sentry_sdk/integrations/quart.py

```diff
@@ -116,18 +115,19 @@ def decorator(old_func):
                 @wraps(old_func)
                 def _sentry_func(*args, **kwargs):
                     # type: (*Any, **Any) -> Any
-                    hub = Hub.current
-                    integration = hub.get_integration(QuartIntegration)
+                    integration = sentry_sdk.get_client().get_integration(
+                        QuartIntegration
+                    )
                     if integration is None:
                         return old_func(*args, **kwargs)
 
-                    with hub.configure_scope() as sentry_scope:
-                        if sentry_scope.profile is not None:
-                            sentry_scope.profile.active_thread_id = (
-                                threading.current_thread().ident
-                            )
+                    scope = Scope.get_isolation_scope()
+                    if scope.profile is not None:
+                        scope.profile.active_thread_id = (
+                            threading.current_thread().ident
+                        )
```

Comment:
Should this threading stuff go to the isolation scope? Or would current scope make more sense?

Reply:
I guess isolation scope is fine. Other integrations also do it on the isolation scope (ex fastapi). One request will probably be handled by on thread in quart.